pkg/controllers/restore/controller.go

```diff
@@ -477,6 +476,26 @@ func (h *handler) generateDependencyGraph(ownerToDependentsList map[string][]res
 	return nil
 }
 
+// customize provides customization of restored resource for edge cases
+func customize(obj *unstructured.Unstructured) {
+	switch obj.GetKind() {
+	case "ServiceAccount":
+		// remove secrets section as referenced secrets will be removed by k8s Token Controller as they are considered orphaned
+		delete(obj.Object, secretsMapKey)
+		logrus.Debugf("Secrets section is ignored in ServiceAccount %s/%s", obj.GetNamespace(), obj.GetName())
+	case "Cluster":
+		// for fleet cluster it needs to be reimported in order to reissue service account token that is no longer valid
+		if obj.GetAPIVersion() == "fleet.cattle.io/v1alpha1" {
+			n, err := rand.Int(rand.Reader, big.NewInt(99999))
```

Comment:
I know it's one of those "never going to happen", but technically `err` may be non-nil and we simply ignore it here with no logging or actual error.

Reply:
I am going to take +1 approach here instead of random int, seems overkill.